sdk/iot/modelsrepository/src/index.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft.
+// Licensed under the MIT license.
+
+/**
+ * This is the ModelsRepositoryClient Library for Javascript.
+ * Why Models and not Model? Because there's more than one Model.
+ *
+ * @remarks
```

Comment:
honestly not sure! @packageDocumentation was something I found out about when, while generating the API docs, I found that it would add a line like this (seen in all other repos it looks like): 

```js
// (No @packageDocumentation comment for this package)
```

So I added packageDocumentation to make it go away.

Reply:
Huh, I'm not opposed to it, but I wonder if we can leverage it in the ref docs so your work doesn't go unnoticed